src/graph.rs

```diff
@@ -2648,7 +2669,7 @@ fn analyze_dynamic_arg_template_parts(
 
 /// Determine if a media type is "untyped" and should be checked to see if there
 /// are types provided.
-fn is_untyped(media_type: &MediaType) -> bool {
+pub fn is_untyped(media_type: &MediaType) -> bool {
```

Comment:
Maybe we should change this to something like `is_untyped_script` or something like that (or switch to `is_typed_media_type`). Btw, `MediaType` is copy so we can change this to just `MediaType`.